packages/connection/tests/php/test-class-nonce-handler.php

```diff
@@ -86,7 +86,7 @@ public function test_add_existing() {
 		$query_filter = function( $result, $query ) use ( &$query_filter_run, $nonce_object ) {
 			if ( ! $query_filter_run && false !== strpos( $query, 'jetpack_nonce_' ) ) {
 				$query_filter_run = true;
-				$this->assertEquals( "SELECT * FROM `options` WHERE option_name = '{$nonce_object['option_name']}'", $query );
+				$this->assertEquals( "SELECT 1 FROM `options` WHERE option_name = '{$nonce_object->option_name}'", $query );
 
 				return array( $nonce_object );
```

Comment:
Now with the changed query, the return should be something along the lines of

`return array( (object) array( 1 => '1' ) );`

to match what the actual DB will return.

Reply:
Fixed 👍